jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/impl/JettyClientUpgradeRequest.java

```diff
@@ -53,15 +50,7 @@ public JettyClientUpgradeRequest(WebSocketCoreClient coreClient, UpgradeRequest
             request.getHeaders().forEach(fields::put);
```

Comment:
Uh, no. It's a mistake that `request.getHeaders()` returns a `HttpFields.Mutable`.
I'll fix `jetty-10.0.x` so please rebase this PR.
This code should be: `headers(fields -> request.getHeaders().forEach(fields::put))`.